cmd/destroy/destroy.go

```diff
@@ -189,23 +193,6 @@ If you need to destroy external resources (like s3 buckets or other Cloud resour
 				return oktetoErrors.ErrContextIsNotOktetoCluster
 			}
 
-			cwd, err := os.Getwd()
-			if err != nil {
-				return fmt.Errorf("failed to get the current working directory: %w", err)
-			}
```

Comment:
You don't have to remove this. In line 172, there might be a change in the working directory of the command, so `initialWd` might not be the correct one you have to pass to `setOptionsNameAndManifestName`

Reply:
reverted this change